app/src/main/java/com/mobileappdevelopment/themunichquiz/MainActivity.kt

```diff
@@ -7,6 +7,6 @@ class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        setContentView(R.layout.activity_settings)
```

Comment:
This has to be reverted to activity_main
```suggestion
        setContentView(R.layout.activity_main)
```